uri.use

```diff
@@ -0,0 +1,8 @@
+Name: ooc uri
+Description: uri parser
+SourcePath: source/uri
+Imports: Locator
+Imports: Authority
+Imports: User
+Imports: Endpoint
+Imports: Query
```

Comment:
No it wasn't. There is an extension for this, but I would not recommend installing it at this point.